services/app-api/src/emailer/emailer.ts

```diff
@@ -186,6 +195,27 @@ function newSESEmailer(config: EmailConfiguration): Emailer {
                 return await this.sendEmail(emailData)
             }
         },
+        sendQuestionsStateEmail: async function (
```

Comment:
as you are working on these four emails, I wonder if we could refactor the SESEmailer and the localEmailer code to be a little less repetitive